### PR TITLE
Feature sb 74 public product page

### PR DESCRIPTION
**Pull request by ValentinaTenus**
1added product page in two variants. first for buyers(without functiontity to add to cars, to wishlist)
2. second in preview mode for vendors
3. if vendor click anywhere on page modal window with message apears.
1. product page for buyers
image
image
image

4. product page in preview mode for vendor

image
image

image

### CODE REVIEW

File: src/pages/product/components/product-status-radio/styles.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		height: "100%",
+	},
+	"& .MuiRadio-root span": {
+		"&:hover": {
+			backgroundColor: "transparent",
+		},
+		backgroundColor: "transparent",
+	},
+	background: "white",
+	border: "1px solid #EDEAE9",
```
Comment (VladManukian) [inline src/pages/product/components/product-status-radio/styles.ts]: use const color insted of EDEAE9 or "white"

File: src/pages/product/components/radio-label/styles.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const StyledRadioLabelContent = styled(Box)`
+	display: flex;
+	flex-direction: column;
+	gap: 4px;
+	height: 100%;
+	width: 100%;
+`;
+
+const StyledRadioLabelImage = styled(Box)({
+	border: "1px solid #EDEAE9",
```
Comment (VladManukian) [inline src/pages/product/components/radio-label/styles.ts]: Use const instead of EDEAE9

**Review by VladManukian — APPROVED** [review]

**Review by jaroslawtt — COMMENTED** [review]
conflicts

**Review by likeri29 — APPROVED** [review]

**Comment by VladManukian** [comment]
1. Size should be in upper case 
And two small questions: 
1. Why colors other for this items? It is expected? 
2. And what does mean height? We use international format for size aren't we?
productfeed

**Comment by ValentinaTenus** [comment]
> 1. Size should be in upper case
>    And two small questions:
> 2. Why colors other for this items? It is expected?
> 3. And what does mean height? We use international format for size aren't we?
>    productfeed

sizes and colors from seeds and I think right of them will be added in adding product task. about height I just made as in figma. for now our product doesn't have height.

**Review by VladManukian — APPROVED** [review]

**Pull request merged**